### PR TITLE
couponapp

### PR DESCRIPTION
**Pull request by gamerson**
- add links and delete extra configmaps
- fix the docker binds from gradle task
- new coupon-app
- Revert "use an explicit token request as recommended in later kubernetes versions"
- fix service account token usage
- rename custom element
- some changes to cors to get demo to work

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+  hostConfig.binds.putAll(["liferayData":"/opt/liferay/data:rw"])
   hostConfig.network = "k3d-lxc-localdev"
 
-  withEnvVar "KUBERNETES_SERVICE_HOST", "k3d-lxc-localdev-server-0"
-  withEnvVar "KUBERNETES_SERVICE_PORT", "6443"
-  withEnvVar "KUBERNETES_NAMESPACE", "default"
+	withEnvVar "KUBERNETES_SERVICE_HOST", "k3d-lxc-localdev-server-0"
+	withEnvVar "KUBERNETES_SERVICE_PORT", "6443"
+	withEnvVar "KUBERNETES_NAMESPACE", "default"
+
```
Comment (rotty3000) [inline build.gradle:8]: I guess we don't need this formatting change.

Comment (rotty3000) [reply build.gradle:8]: but not super important.

File: extensions/couponpdf/src/main/java/com/liferay/couponpdf/service/config/HttpSecurityConfig.java [hunk 2]
```diff
@@ -10,5 +10,6 @@ public class HttpSecurityConfig extends WebSecurityConfigurerAdapter {
   @Override
   protected void configure(HttpSecurity httpSecurity) throws Exception {
     httpSecurity.csrf().disable().authorizeRequests().antMatchers("/").permitAll();
+    httpSecurity.cors();
   }
 }
```
Comment (rotty3000) [inline extensions/couponpdf/src/main/java/com/liferay/couponpdf/service/config/HttpSecurityConfig.java:15]: we should have a cors pattern in the original springboot app... we need to dig it up. But we also have new properties to load the correct domains from too... we'll tune this.

**Review by rotty3000 — APPROVED** [review]
LGTM

File: extensions/couponpdf/src/main/java/com/liferay/couponpdf/service/config/HttpSecurityConfig.java (hunk 2 above)
Comment (gamerson) [reply extensions/couponpdf/src/main/java/com/liferay/couponpdf/service/config/HttpSecurityConfig.java:15]: Right, I also had to modify the domains to `*` which is wrong.  Maybe you can help me investigate.

**Pull request merged**